src/shogun/classifier/Perceptron.cpp

```diff
@@ -35,7 +35,9 @@ void CPerceptron::init()
 	max_iter = 1000;
 	learn_rate = 0.1;
 	m_initialize_hyperplane = true;
-	SG_ADD(&max_iter, "initialize_hyperplane", "Whether to initialize hyperplane.", MS_AVAILABLE);
```

Comment:
why the change? just by editor accident?

Reply:
see that its &max_iter and not &m_initialize_hyperplane

Reply:
ah yes, correcting copy paste errors. nice!